manager/src/main/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPI.java

```diff
@@ -34,6 +41,7 @@
 
 import static java.util.stream.Collectors.toList;
 
+@Tag(name = "Shard API", description = "The API that allow a shard to retrieve and update resources.")
```

Comment:
@r00ta hello,

for our ticket https://issues.redhat.com/browse/MGDOBR-420, it would be beneficial, if the `@Tag`s we added as part of this PR, would omit the `API` name suffix. So for example, would it be possible to have `Shard` instead of `Shard API`?

For the MGDOBR-420 ticket it would help to have nice generated file names. Currently we have like `bridges-apiapi.ts` with the suggested change we would get `bridges-api.ts`. Can I open a PR?